cluster/examples/vm-template-rhel7.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     tags: kubevirt,ocp,template,linux,virtualmachine
   creationTimestamp: null
   labels:
-    kubevirt.io/os: rhel-7.4
+    kubevirt.io/os: rhel7.4
```

Comment:
why is the preset for a specific minor version vs. just rhel7?

Reply:
I meant to answer that earlier.

`kubevirt.io/os` takes a libosinfo identifier as a value. And there minor versions have identifiers, not the major version.